apps/web/src/pages/quick-start/steps/Setup.tsx

```diff
@@ -146,8 +148,14 @@ const LoaderWrapper = styled.div`
   margin-top: 10px;
 `;
 
-function updateCodeSnipped(codeSnippet: string, environmentIdentifier: string) {
-  return codeSnippet.replace(APPLICATION_IDENTIFIER, environmentIdentifier);
+function updateCodeSnipped(codeSnippet: string, environmentIdentifier: string, apiKey: string) {
```

Comment:
```suggestion
function updateCodeSnippet(codeSnippet: string, environmentIdentifier: string, apiKey: string) {
```
Typo.

Reply:
Good catch! I do it all the time, but not always catch it 🤦‍♂️